togetherly/lib/models/reward.dart

```diff
@@ -6,13 +8,15 @@ class Reward {
   final String description;
   final int points;
   final int quantity;
+  final Char icon;
```

Comment:
I will push a change to make this an `int`, rather than a `Char`, so that we are not limited to single-byte characters.